Packs/Rapid7_Nexpose/ReleaseNotes/1_2_15.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Vulnerability Handling - Nexpose
+
+- The sub-playbooks "CVE Enrichment - Generic" and "Calculate Severity - Generic" is no longer in use and has been replaced by the sub-playbooks "CVE Enrichment - Generic v2" and "Calculate Severity - 3rd-party integrations". The old playbook will still be available for six months, after which it will be removed from the marketplace.
```

Comment:
remove this line "The old playbook will still be available for six months, after which it will be removed from the marketplace."